### ISSUE TITLE
Support Scaffolding SJS Transforms

### ISSUE THREAD

**Issue by RobertSzkutak** [marklogic/roxy#682]
Currently at least one customer asking for this feature

**Comment by grtjn** [comment]
Has anyone looked yet at how much effort this would be?

**Comment by RobertSzkutak** [comment]
Seemed pretty minimal. We already have a pattern in place for XQuery we could adopt.

**Comment by grtjn** [comment]
Yeah maybe, forgot how intelligent that scaffold logic was, but maybe it is dead-simple. We would most importantly need a few good code samples..
